apps/documentation/docs/fresco/deployment/advanced.en.mdx

```diff
@@ -1,7 +1,267 @@
+    image: postgres:16-alpine
+    restart: always
+    env_file:
+      - .env
+    volumes:
+      - postgres:/var/lib/postgresql/data
+    healthcheck:
+      test: ['CMD', 'pg_isready', '-U', 'postgres']
+      interval: 5s
+      timeout: 10s
+      retries: 5
+    networks:
+      - fresco_network
+
+volumes:
+  postgres:
+
+# Define a network, which allows containers to communicate
+# with each other, by using their container name as a hostname
+networks:
+  fresco_network:
+```
+
+### Build and Run Fresco with Docker Compose
+
+Start up your application by running the following command:
+
+```bash
+docker-compose up
+```
+
+## Access Fresco
+
+You can now access Fresco by visiting the `PUBLIC_URL` that you have configured in your hosting service.
+
+## Configuration and setup
+
+To use Fresco, you need to set up an administrator account which will enable you to access the protected parts of the app for study management. Only one administrator account can be created.
+
+<TipBox danger>
+
+To ensure the security of your deployment, you have **30 minutes** from when the app is deployed to create a user account. If this time elapses without a user account being created, your configuration will expire. To redeploy, follow our guide to [reset your database](/en/fresco/deployment/troubleshooting#instance-expired-message-during-the-setup-process).
+
+</TipBox>
+
+### Create a user account
+
+Visit your deployed app and create an account by entering a username and password. Ensure you make the password as strong as possible!
+
+<TipBox>
+
+Only **one** user account can be created in Fresco. If you need access for multiple team members, we recommend using a shared username and password for the administrator account to ensure that your team can access the platform.
+
+</TipBox>
+
+![Enter a username and password](/assets/img/fresco-images/fresco1.png)
+
+<TipBox danger>
+
+If you forget your password, you **will not be able to reset it**. We recommend keeping a record of your username and password in a secure location.
+
+</TipBox>
+
+### Upload a protocol
+
+You can optionally upload a protocol. Protocols can also be uploaded from the protocols page once the setup steps are completed.
+
+![Upload a protocol](/assets/img/fresco-images/fresco2.png)
+
+### Configure participation
+
+- **Import participants**: You can optionally upload a CSV file containing participants you wish to recruit to your study. Participants can also be uploaded from the participants page once the setup steps are completed.
+
+- **Allow anonymous recruitment**: You can enable this setting to allow your participants to join your study by visiting a URL. More information about anonymous recruitment can be found in the [using Fresco](/en/fresco/using-fresco) guide.
+
+![Import participants and optionally allow anonymous recruitment](/assets/img/fresco-images/fresco3.png)
+
+### Visit the dashboard
+
+Click the **"Go to the dashboard"** button and to begin using Fresco!
```

Comment:
Replace this with a link to the 'configration' section of the basic deployment article, so that we don't have to duplicate: https://documentation-oaelb1noe-network-canvas-f4790d84.vercel.app/en/fresco/deployment/guide#configuration-and-setup